### ISSUE TITLE
Collecting blood samples from bots doesn't count towards traitor mission completion

### ISSUE THREAD

**Issue by Rokvach** [Regalis11/Barotrauma#12686]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

![image](https://github.com/Regalis11/Barotrauma/assets/7299826/e5494d91-534d-43e3-9934-f283e548de18)

### Reproduction steps

-

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

Other

### -

Unstable V1.1.13.0

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Could not reproduce. :/ Tested by starting a multiplayer round with 2 bots, and the objective seems to complete when I've collected a sample from both:
![image](https://github.com/Regalis11/Barotrauma/assets/14351070/812a556d-1c7b-4509-9c4b-b5162594032f)

It's worth noting that filling the vials is not the objective, you need to get a sample from everyone. Are you sure you got one from every other character (bots and human players)?

**Comment by Rokvach** [comment]
Yeah I collected the blood mostly from the same bots as I didn't know you need to collect it from everyone, I think we can close this but I also think we need to take a closer look at this later with GA data as currently I don't see the mission being that completable.
